packages/gatsby-theme-docs/src/layouts/internals/reset.js

```diff
@@ -0,0 +1,358 @@
+ * from bootstrap's "reboot", which is a fork off normalize.css.
+ * It does not preformat semantic content elements, this is done in the styled components.
+ */
+const Reset = () => (
+  <>
+    <Global
+      styles={css`
+        html,
+        body,
+        #___gatsby,
+        #gatsby-focus-wrapper {
+          height: 100%;
+          width: 100%;
+        }
+        html,
+        div,
+        span,
+        applet,
+        object,
+        iframe,
+        h1,
+        h2,
+        h3,
+        h4,
+        h5,
+        h6,
+        p,
+        blockquote,
+        pre,
+        a,
+        abbr,
+        acronym,
+        address,
+        big,
+        cite,
+        code,
+        del,
+        dfn,
+        em,
+        img,
+        ins,
+        kbd,
+        q,
+        s,
+        samp,
+        small,
+        strike,
+        strong,
+        sub,
+        sup,
+        tt,
+        var,
+        b,
+        u,
+        i,
+        center,
+        dl,
+        dt,
+        dd,
+        ol,
+        ul,
+        li,
+        fieldset,
+        form,
+        label,
+        legend,
+        table,
+        caption,
+        tbody,
+        tfoot,
+        thead,
+        tr,
+        th,
+        td,
+        article,
+        aside,
+        canvas,
+        details,
+        embed,
+        figure,
+        figcaption,
+        footer,
+        header,
+        hgroup,
+        menu,
+        nav,
+        output,
+        ruby,
+        section,
+        summary,
+        time,
+        mark,
+        audio,
+        video {
+          margin: 0;
+          padding: 0;
+          border: 0;
+          font-size: 100%;
+          vertical-align: baseline;
+          min-width: 0; /* flexbox defaults to auto causing confusion*/
+        }
+        /* HTML5 display-role reset for older browsers */
+        article,
+        aside,
+        details,
+        figcaption,
+        figure,
+        footer,
+        header,
+        hgroup,
+        menu,
+        nav,
+        section {
+          display: block;
+        }
+        ol,
+        ul {
+          list-style: none;
+        }
+        blockquote,
+        q {
+          quotes: none;
+        }
+        blockquote:before,
+        blockquote:after,
+        q:before,
+        q:after {
+          content: '';
+          content: none;
+        }
+        table {
+          border-collapse: collapse;
+          border-spacing: 0;
+        }
+
+        /* Change from 'box-sizing: content-box' so that 'width' is not affected by 'padding' or 'border'. */
+        *,
+        *::before,
+        *::after {
+          box-sizing: border-box;
+        }
+
+        body {
+          margin: 0;
+          padding: 0;
+          border: 0;
+          font-style: normal;
+          font-variant: normal;
+          font-stretch: normal;
+          font-family: 'Roboto', -apple-system, system-ui, BlinkMacSystemFont,
```

Comment:
We can probably remove the font styling here, since we define it in the `Global` component already.
We should only have one source of truth.